web/themes/joinup/templates/custom-form.html.twig

```diff
@@ -0,0 +1,16 @@
+{#
+/**
+ * @file
```

Comment:
> What is the purpose of this custom form?

It allows to separate content and action buttons

> As it seems a bare template, can a template be avoided?

It is in fact only a template for content which is between <form> tags, so other containers are not needed there, because <form> tags are still there.

> Can you think of a better naming?

Hmm, this is a name for all forms which have action buttons separated form the content. Maybe 'buttons-separated-form'?